uui-components/src/table/DataTableRowContainer.tsx

```diff
@@ -31,16 +31,16 @@ function getSectionStyle(columns: DataColumnProps[], minGrow = 0) {
     let width = 0;
 
     columns.forEach(column => {
-        const columnWidth = (typeof column.width === 'number' ? column.width : (column.minWidth || 0));
+        const columnWidth = (typeof column.width === 'number' ? (column.fix ? column.width : column.width - 1) : (column.minWidth || 0));  // (column.width - 1) do not forget the negative margin of the scrolling columns in the calculation of the width
```

Comment:
move border width value into const variable and use it here and assign it to css-variable